architecture/platforms/versal_acap.rst

```diff
@@ -0,0 +1,125 @@
+.. _versal_acap:
+
+#############################################
+VERSAL Adaptive Compute Acceleration Platform
+#############################################
+
+
+.. _overview :
+
+Overview
+********
+
+The OP-TEE support for the AMD/Xilinx Versal Adaptive Compute Acceleration Platform (`Versal ACAP`_) delegates most of its functionality to the `PLM firmware`_ executing in the MicroBlaze processor.
```

Comment:
Most other files wrap lines at ~80 characters which I think is better when reading in source format in a text editor (code blocks being an exception).